### DISCUSSION TITLE
Geomap - Lines - Polygon

### DISCUSSION THREAD

**Discussion by j-dc** [grafana/grafana#53507] [Dashboards, Panels & Visualizations]
Hey,

It would be really great to have a 'Lines' and  a 'Polygons' layer on the geomap.

This could be easily done by adding a 'metric' or 'id' field next to, lattitude and longitude field.
Every metric/id would result in one line/polygon.

This would be a realy a great addition to an otherwise powerfull tool!

I'd love to hear your opinions about this.

**Comment by bocajim** [comment]
I agree this would be very useful, I would like to find a way to draw simply polygons (to represent geofences).

**Comment by MKloberg** [comment]
Hi! 
 Don't laugh, but my dog has his own dashboard and I'm tracking his GPS location, plotting that with the geomap.
It would be nice if there was a way to connect the dots with lines and of course also plot where the geo-fence is that sets off the alert when he goes astray too far from the house.

I could think of a great many other ways to use this feature request at work if it gets implemented.

![image](https://user-images.githubusercontent.com/13971502/204114389-a45c322c-639a-4d7a-83b7-a883e9c76cb2.png)

Thanks,
Mac

**Comment by JPFreeski** [comment]
I would also +1 on this feature as I've got a course that I want to draw lines between points on and make the lines be able to show red for 0 or green for 1.

Please Grafana I love you.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
